doc: removed hierarchyBone, changed to Python 3.10+ and removed wiki and community landing page (#707)
viur-framework/viur-core: docs/doc_start/15_basics.rst

That's it. When this Skeleton is connected to a module later on, ViUR's admin tools like the Vi automatically provide an auto-generated input mask on it.

> A Skeleton does automatically provide the bone ``key`` also, which is an instance of the class :class:`keyBone<core.bones.bone.keyBone>`. This bone holds the value of the unique entity key, that is required to uniquely identify an entity within the database. The pre-defined bones ``creationdate`` and ``changedate`` of each skeleton store the date and time when the entity was created or changed. In terms of ViUR, an entity is a document or dataset in the datastore, that stores information.

Reviewer: Use upper-case class names only. It's now `KeyBone`, `BaseBone`, `StringBone`, etc.

> A Skeleton does automatically provide the bone ``key`` also, which is an instance of the class :class:`keyBone<core.bones.bone.keyBone>`. This bone holds the value of the unique entity key, that is required to uniquely identify an entity within the database. The pre-defined bones ``creationdate`` and ``changedate`` of each skeleton store the date and time when the entity was created or changed. In terms of ViUR, an entity is a document or dataset in the datastore, that stores information.

Reviewer 2: ```suggestion
A Skeleton does automatically provide the bone ``key`` also, which is an instance of the class :class:`KeyBone<core.bones.bone.key>`. This bone holds the value of the unique entity key, that is required to uniquely identify an entity within the database. The pre-defined bones ``creationdate`` and ``changedate`` of each skeleton store the date and time when the entity was created or changed. In terms of ViUR, an entity is a document or dataset in the datastore, that stores information.
```

Prototypes and modules
======================

> Modules are the controllers of a ViUR application, and implement the application logic. To implement modules, ViUR provides three basic prototypes. These are :class:`List<core.prototypes.list.List>`, :class:`Singleton<core.prototypes.singleton.Singleton>` and :class:`Tree<core.prototypes.tree.Tree>`.

Reviewer 2: It is true that we no longer have hierarchy. But for this we have now the `SkelModule`.

```suggestion
Modules are the controllers of a ViUR application, and implement the application logic. To implement modules, ViUR provides four basic prototypes. These are :class:`List<core.prototypes.list.List>`, 
:class:`Singleton<core.prototypes.singleton.Singleton>`,
:class:`Singleton<core.prototypes.skelmodule.SkelModule> and :class:`Tree<core.prototypes.tree.Tree>`.
```

- :class:`User<core.modules.user.User>` implements a user login, authentication and management module,

> - :class:`Cart<core.modules.cart.Cart>` and :class:`Order<core.modules.order.Order>` implement modules for submitting and managing orders in a web-shop,

Reviewer 2: This module has been removed

```suggestion
```
